lib/index.js

```diff
@@ -74,17 +74,10 @@ export default {
 
         // Call ERB to "de-templatize" the code
         return helpers.exec(this.erbPath, erbArgs,
-          { stdin: text, cwd: fileDir }
+          { stdin: text.replace(/<%=/g, '<%'), cwd: fileDir }
```

Comment:
Ok, found an answer. The `language-ruby-on-rails` package isn't marking Rails ERB files as such, but just as regular ERB files. Until [this issue](https://github.com/atom/language-ruby-on-rails/issues/14) is fixed we have to leave `linter-erb` as is, otherwise we would be back in the situation where this package was raising errors on the rails syntax.

When that issue is fixed then, the string replacement introduced with this commit should be limited to only the `text.html.ruby` scope.